### DISCUSSION TITLE
Is importing the Server class from main intentional?

### DISCUSSION THREAD

**Discussion by tback** [Kludex/uvicorn#2527] [Potential Issue] [answered]
I was just browsing through the code and noticed the Server class was imported from main instead of from server. Is this intentional or by accident?

https://github.com/encode/uvicorn/blob/b7241e1c64b8e10f56191d2836fa386ad3e56681/uvicorn/workers.py#L14

(I created this discussion by clicking on the line and referring to it in a discussion, however there was no link to the line in this discussion)

**Comment by Kludex** [comment, marked as answer]
Not intentional, PR welcome.

**Reply by tback** [reply]
ok: https://github.com/encode/uvicorn/pull/2528
